orc8r/cloud/go/services/analytics/analytics/main.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	defaultAnalysisSchedule = "*/1 * * * *" // Every 1 minute
+	defaultAnalysisSchedule = "*/15 * * * *" // Every 15 minutes
```

Comment:
My preference is to *not* have default config values hidden throughout the codebase. Instead, use the configs checked into vcs to hold the default values. This increases visibility into config values as the numberr of config values increases